docs/DeveloperGuide.md

```diff
@@ -281,18 +281,40 @@ The following sequence diagram shows how the viewTransactionsByStatus operation
 
 ### Target user profile
 
-{Describe the target user profile}
+- Computing student at NUS
+- Reasonably comfortable using CLI
+- Student staying at campus
+- Has a passion to serve the student community
+- Student that have items he does not require
+- Student that wants to earn some money
+- Student that wants to borrow an item he does not own
```

Comment:
I think the users are not the students who have items or want to borrow smt, but the student who wants to host a rental business that will helps others to put their items on the list for rent